testsuite/MDAnalysisTests/coordinates/data/create_data.py

```diff
@@ -0,0 +1,39 @@
+import MDAnalysis as mda
+import numpy as np
+from six.moves import range
+
+
+def create_test_trj(uni, fname, wrong_natoms=False):
+    n_atoms = uni.atoms.n_atom
+    if wrong_natoms:
+        n_atoms -= 2
+    pos = np.arange(3 * n_atoms).reshape(n_atoms, 3)
+    uni.trajectory.ts.dt = 1
+    with mda.Writer(fname, n_atoms) as w:
```

Comment:
I don't think that it's that simple in practice, especially if it's a format that one is not full familiar with or that is not even well specified (like many!). Ultimately, MDAnalysis must be able to cope with what's in the wild to be useful and in the past it was pretty useful to work with real data instead of synthetic one. Nevertheless, this is a worthwhile discussion to be had but we should probably not have in a line note but a separate issue ;-).